pkg/operator/render.go

```diff
@@ -123,6 +123,9 @@ func createDiscoveredControllerConfigSpec(infra *configv1.Infrastructure, networ
 			glog.V(2).Info("Not setting proxy config because Proxy status is empty")
 		} else {
 			ccSpec.Proxy = &proxy.Status
+			// Not doing this will result into LINE_MAX limit errors from the ignition transpiler
+			// see https://bugzilla.redhat.com/show_bug.cgi?id=1898169
+			ccSpec.Proxy.NoProxy = fmt.Sprintf("%s", strings.Join(strings.Split(ccSpec.Proxy.NoProxy, ","), ",\\\n"))
```

Comment:
To be safe here perhaps we should add a nil check for NoProxy?

Reply:
sorry, changed my mind and re-pushed what I think it's a better fix (noProxy is a string tho :thinking: )